docs/body.md

```diff
@@ -459,6 +459,16 @@ This ZenPack still supports the previous configuration, where messages were sent
 
 For new openstack installs, we recommend using the RabbitMQ-Ceilometer endpoint, which is what will be reported by `openstack_amqp_config`.
 
+When the RabbitMQ-Ceilometer service is added to Control Center, two IP Assignments will be added as well.  Upon creation, they will have no IP address, only a port number.  You must add the IP address by following these steps:
```

Comment:
I think if the ZP is installed before a collector is deployed, the IP assignment will be "automatic" but if installed after deployment you'll need to set it, but I don't really recall.  I agree with Josh that we should have them check the assignments and set/change them if neccessary.  Also, the instructions should probably reference the assignments explicitely instead of "the first".  Perhaps describe what the 2 assignments are for and what their names/ports will be?